malwarecage/web/src/styles/index.css

```
@@ -241,4 +241,12 @@ a.blob {
 /* react-modal css fix */
 .ReactModal__Overlay {
     z-index: 3;
-}
\ No newline at end of file
+}
+
+.flicker-element td i {
+    visibility: hidden;
+}
+
+.flicker-element td:hover i {
+    visibility: visible;
+}
```

I would add these two to have better interaction feedback
```suggestion
}

.flicker-element td i:hover {
    color: var(--blue);
}

.flicker-element td i:active {
    color: var(--red);
}
```